tldraw/tldraw
# Fix fontStyle assignment

### Change type


### Test plan

1. Create a text
2. Export as SVG
3. Check that `<text>` element has a valid `font-style` attribute value


### Release notes

- Fixed a bug where font style wasn't correctly exported as SVG
